src/CreateEntityMappingView/CreateEntityMappingView.js

```diff
@@ -0,0 +1,319 @@
+		const mappings = this.state.mappings;
+		mappings.push({
+			entityTypeAttribute: {value: null, errorMessage: ""},
+			eventTypeAttribute: {value: null, errorMessage: ""}
+		});
+		this.setState({
+			mappings: mappings
+		});
+	}
+
+	handleEventTypeAttributeChange(key, eventTypeAttribute) {
+		const mappings = this.state.mappings;
+		mappings[key].eventTypeAttribute.value = eventTypeAttribute;
+        mappings[key].eventTypeAttribute.errorMessage = "";
+		this.setState({
+			mappings: mappings
+		});
+	}
+
+	handleEntityTypeAttributeChange(key, entityTypeAttribute) {
+		const mappings = this.state.mappings;
+		mappings[key].entityTypeAttribute.value = entityTypeAttribute;
+        mappings[key].entityTypeAttribute.errorMessage = "";
+		this.setState({
+			mappings: mappings
+		});
+	}
+
+	getEventTypeAttributesDropDown(key, selectedEventTypeAttribute) {
+		const eventTypeAttributes = this.props.eventTypeAttributes.value;
+		return (<SelectField
+			floatingLabelText="Select Event Type Attribute"
+			fullWidth={true}
+			errorText={this.state.mappings[key].eventTypeAttribute.errorMessage}
+			value={selectedEventTypeAttribute}
+			onChange={(event, index, value) => {
+				this.handleEventTypeAttributeChange(key, value);
+			}}>
+			{CreateEntityMappingView.getMenuItems(eventTypeAttributes)}
+		</SelectField>);
+	}
+
+    getEntityTypeAttributesDropDown(key, selectedEntityTypeAttribute) {
+        const entityTypeAttributes = this.props.entityTypeAttributes.value;
+        return (<SelectField
+			floatingLabelText="Select Entity Type Attribute"
+			fullWidth={true}
+			value={selectedEntityTypeAttribute}
+			errorText={this.state.mappings[key].entityTypeAttribute.errorMessage}
+			onChange={(event, index, value) => {
+                this.handleEntityTypeAttributeChange(key, value);
+            }}>
+            {CreateEntityMappingView.getMenuItems(entityTypeAttributes)}
+		</SelectField>);
+    }
+
+	isValidInput() {
+		let isValid = true;
+		if(!this.state.selectedEntityTypeId.value) {
+			this.setState({
+				selectedEntityTypeId: {value: null, errorMessage: config.messages.requiredFieldMessage}
+			});
+			isValid = false;
+        }
+        if (!this.state.selectedEventTypeId.value) {
+            this.setState({
+                selectedEventTypeId: {value: null, errorMessage: config.messages.requiredFieldMessage}
+            });
+            isValid = false;
+		}
+		let mappings = [];
+        this.state.mappings.forEach((mapping) => {
+			if (!mapping.eventTypeAttribute.value) {
+				mapping.eventTypeAttribute.errorMessage = config.messages.requiredFieldMessage;
+				isValid = false;
+            }
+            if (!mapping.entityTypeAttribute.value) {
+                mapping.entityTypeAttribute.errorMessage = config.messages.requiredFieldMessage;
+				isValid = false;
+			}
+			mappings.push(mapping);
+		});
+        this.setState({
+            mappings: mappings
+        });
+		console.log(isValid);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)